agent/lib_guzzle6.c

```diff
@@ -65,12 +65,11 @@
  */
 #if ZEND_MODULE_API_NO >= ZEND_5_5_X_API_NO
 
-/* {{{ newrelic\Guzzle6\RequestHandler class definition and methods */
-
 /*
  * True global for the RequestHandler class entry.
  */
 zend_class_entry* nr_guzzle6_requesthandler_ce;
+int php_version_compare(char*, char*);
```

Comment:
This seems unnecessary. Instead, `#include "ext/standard/php_versioning.h"` should be added to the `#include` statements

```suggestion

```